scripts/remove/calibrecs.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+#shellcheck source=sources/functions/utils
+. /etc/swizzin/sources/functions/utils
+
+if [ -z "$CALIBRE_LIBRARY_USER" ]; then
+    if ! CALIBRE_LIBRARY_USER="$(swizdb get calibre/library_user)"; then
+        CALIBRE_LIBRARY_USER=$(_get_master_username)
+        swizdb set "calibre/library_user" "$CALIBRE_LIBRARY_USER"
```

Comment:
I'm having trouble trying to understand why we're setting this variable in swizdb while we are removing the app.

If a user removed calibre first then this would leave your install with swizdb cruft.

Reply:
> I'm having trouble trying to understand why we're setting this variable in swizdb while we are removing the app.

Yeah me too lol